packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -151,6 +176,17 @@ test('zoe - coveredCall', async t => {
               `Bob didn't get anything back`,
             ),
           );
+
+        await E(seat)
```

Comment:
API design issue: This whole pattern with `getPayout` with a named payout that could be wrong is scary. It seems very easy in many usages to drop a payment on the floor.  We need some mechanism or pattern to prevent that.

Reply:
Hmm, that's true. I'll turn this comment into an issue.

Reply:
Resolving this as it has been made into an issue